### ISSUE TITLE
Confusion about passing the test dataset

### ISSUE THREAD

**Issue by Med-Rokaimi** [FateMurphy/CEEMDAN_LSTM#13]
Hey,
Thank you so much for the greate package.
I'm confusing about the test step. I mean I will pass the train dataset for the model (eg. df_result = kr.hybrid_keras_predict(data=series, show=True, plot=True, save=True))
How to pass my test dataset after that after training phase?

**Comment by FateMurphy** [comment]
The data contains both the training set and the test set because this package has a special way of splitting the data. If you want to predict step by step and change the splitting way, you can refer to cl.show_keras_example() or https://github.com/FateMurphy/CEEMDAN-VMD-GRU
